app/decorators/reviewable.rb

```diff
@@ -5,6 +5,15 @@ def class
     __getobj__.class
   end
 
+  def attributes=(attributes)
+    if attributes["reviewed"] == "1"
```

Comment:
Let's rename this attribute to "needs_review" or something similar.  This is very confusing otherwise, and can't possibly lead to anything good.  Especially since the attribute doesn't match the method.